### ISSUE TITLE
Initiative: Visualize masking before pretraining/finetuning

### ISSUE THREAD

**Issue by wael-mika** [ecmwf/WeatherGenerator#2145]
### Describe the task. Describe the task. It can be a feature, a set of experiments, documentation, etc.

As many team members requested to have this feature of previewing masked source and target before starting any pretraining or fine-tuning, I am adding a script to provide a preview of how the data would look like before submitting any job. Easy to use script with one line.

### Area

- [x] datasets, data readers, data preparation and transfer
- [ ] model
- [ ] science
- [ ] infrastructure and engineering
- [x] evaluation, export and visualization
- [ ] documentation
- [ ] performance
